addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -824,11 +824,15 @@ registry.backgroundVideo = publicWidget.Widget.extend({
         // YouTube does not allow to auto-play video in mobile devices, so we
         // have to play the video manually.
         if (this.isMobileEnv && this.isYoutubeVideo) {
-            new window.YT.Player(this.iframeID, {
-                events: {
-                    onReady: ev => ev.target.playVideo(),
-                }
+            var iframeID = this.iframeID;
+            window.YT.ready(function(){ 
```

Comment:
Double linter error here, could you please check to install one? I think we provide an Odoo configuration for eslint in the odoo documentation, let me know if you can't find it or need some help

Reply:
https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines :+1: